### ISSUE TITLE
Uncaught FileNotFoundError: [Errno 2] No such file or directory: 'postgresql://XXXXXXXX'
when saving project to Postgresql

### ISSUE THREAD

**Issue by AsgerPetersen** [opengisch/qgis_trackable_project_files#18]
Steps to reproduce
 1. Create project
 2. Save project to postgresql

Traceback with some values replaced:

```
Traceback (most recent call last):
  File "/Users/asger/Library/Application Support/QGIS/QGIS3/profiles/default/python/plugins/trackable_project_files/__init__.py", line 33, in normalize_xml
    with open(fn, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'postgresql://xxx:yyy@zzzz.dk:5432?dbname=aaaa&schema=bbbb&project=cccc'

```
trackable_project_files 1.2.0, QGIS 3.20.0-Odense on Darwin 20.5.0

**Comment by m-kuhn** [comment]
Thanks for reporting. I guess the error message is clear enough.
Not sure what would be involved to fix this for real (i.e. clean the project stored in the database). But possibly just silencing the error would be good enough? You probably have this plugin installed for other projects which are on disk and managed with GIT?

**Comment by AsgerPetersen** [comment]
Yes, I use the plugin for projects stored in qgs-format and managed with git. I would not expect the plugin to try and normalise projects stored in postgres. For my use cases it would be good enough if the plugin silently ignored projects stored in postgres.

**Comment by m-kuhn** [comment]
@AsgerPetersen are you able to verify the implementation in #19?

**Comment by AsgerPetersen** [comment]
I’d be happy to give it a try when I return from summer vacation in about
three weeks 👍

man. 12. jul. 2021 kl. 07.31 skrev Matthias Kuhn ***@***.***>:

**Comment by m-kuhn** [comment]
Enjoy

**Comment by AsgerPetersen** [comment]
I had some trouble reproducing the faulty behaviour. I _think_ it is because my version of the plugin has been upgraded to 1.2.2 since I filed this issue. I guess this commit may have fixed it in 1.2.2: https://github.com/opengisch/qgis_trackable_project_files/commit/1d05c63ea4293d9b7c48d8e14cdcb23f04c89b17

So it seems like the fix was already in the newest plugin version at the time I reported this issue. I am sorry about that!
